doc/01-basic-usage.md

```diff
@@ -0,0 +1,77 @@
+## Basic usage
+
+Castor use a simple convention to find commands. It will look for the first `.castor.php` file in the current directory 
+or in a parent directory.
```

Comment:
```suggestion
or in a parents directory.
```

Reply:
Not sure about the plural here :thinking: